src/lib/core/api/DataClient/types.ts

```diff
@@ -441,16 +441,31 @@ export const LineplotResponse = type({
   completeCasesTable: completeCasesTableArray,
 });
 
-export interface MosaicRequestParams {
+/** JM: open to suggestions on DRY'ing up Mosaic v TwoByTwo parameter interfaces */
+type TwoByTwoRequestConfig = {
+  outputEntityId: string;
+  xAxisVariable: VariableDescriptor;
+  yAxisVariable: VariableDescriptor;
+  facetVariable: ZeroToTwoVariables;
+  showMissingness?: 'TRUE' | 'FALSE';
+  xAxisReferenceValue: string;
+  yAxisReferenceValue: string;
+};
+
+type MosaicRequestConfig = Omit<
+  TwoByTwoRequestConfig,
+  'xAxisReferenceValue' | 'yAxisReferenceValue'
+>;
+
+export interface TwoByTwoRequestParams {
   studyId: string;
   filters: Filter[];
-  config: {
-    outputEntityId: string;
-    xAxisVariable: VariableDescriptor;
-    yAxisVariable: VariableDescriptor;
-    facetVariable: ZeroToTwoVariables;
-    showMissingness?: 'TRUE' | 'FALSE';
-  };
+  config: TwoByTwoRequestConfig;
+}
+
+export interface MosaicRequestParams
+  extends Omit<TwoByTwoRequestParams, 'config'> {
+  config: MosaicRequestConfig;
 }
```

Comment:
Rather than using `Omit`, it might read better to define the Mosaic-related interfaces, and then have the TwoByTwo-related interfaces extend them:

```typescript
interface MosaicRequestConfig {
  outputEntityId: string;
  xAxisVariable: VariableDescriptor;
  yAxisVariable: VariableDescriptor;
  facetVariable: ZeroToTwoVariables;
  showMissingness?: 'TRUE' | 'FALSE';
}

interface TwoByTwoRequestConfig extends MosaicRequestConfig {
  xAxisReferenceValue: string;
  yAxisReferenceValue: string;
};

export interface MosaicRequestParams {
  config: MosaicRequestConfig;
  studyId: string;
  filters: Filter[];
}

export interface TwoByTwoRequestParams extends MosaicRequestParams {
  config: TwoByTwoRequestConfig;
}
```